### PR TITLE
fix(tests): Re-enable Metrics integration test by renaming

### PR DESCRIPTION
**Pull request by ttomsu**
Looks like metrics wasn't working this whole time because this test never got picked up. Whoops...

### CODE REVIEW

**Review by elefeint — APPROVED** [review]

**Review by dzou — APPROVED** [review]

**Review by meltsufin — DISMISSED** [review]
I thought we were using the `*IntegrationTests` pattern for integration tests. Do we just need to list `metrics` as part of the matrix?

**Comment by ttomsu** [comment]
We are - which is why this test results wasn't firing. I was getting a permission denied error in the old repo w GHA and investigated why I wasn't getting it in the new repo. 

The 'metrics' module is already included in the big list of parallelized integration tests.

**Review by elefeint — CHANGES_REQUESTED** [review]

File: docs/src/main/asciidoc/migration-guide-1.x.adoc [hunk 1]
```diff
@@ -84,3 +84,7 @@ Stackdriver Logging::
   ...
 </configuration>
 ----
+
+=== Stackdriver Metrics Explicitly Enabled
+
+If you're importing the `spring-cloud-gcp-starter-metrics` module, the Stackdriver metrics now need to be explicitly enabled with the property `management.metrics.export.stackdriver.enabled=true`
```
[the remark this answers is not in the document]
Comment (ttomsu) [reply docs/src/main/asciidoc/migration-guide-1.x.adoc:90]: I think this property is checked for in Spring Boot, so I don't think we can enable it for the user. Source: https://github.com/spring-projects/spring-boot/blob/master/spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/export/stackdriver/StackdriverMetricsExportAutoConfiguration.java#L50

[the remark this answers is not in the document]
Comment (meltsufin) [reply docs/src/main/asciidoc/migration-guide-1.x.adoc:90]: The fault should be enabled, but it looks like something is setting `management.metrics.export.defaults.enabled=false`. Looking more into it.

[the remark this answers is not in the document]
Comment (meltsufin) [reply docs/src/main/asciidoc/migration-guide-1.x.adoc:90]: It's done here: https://github.com/spring-projects/spring-boot/commit/b1830da0da05c43e67ffc46e5375261e8eb7859a#diff-196296cded479f093a123da98914f159381d3eef89c33f20148075a58085b34fR53

Basically, Spring Boot Test automatically disabled metrics export, unless you add the `@AutoConfigureMetrics` annotation.

I'll update this branch with that.

[the remark this answers is not in the document]
Comment (elefeint) [reply docs/src/main/asciidoc/migration-guide-1.x.adoc:90]: It's ignored in tests, so we need to move the property from `main/resources` to `test/resources`, and no need to document for end users.

**Comment by ttomsu** [comment]
Tests are back to green now.

File: spring-cloud-gcp-samples/spring-cloud-gcp-metrics-sample/pom.xml [hunk 2]
```diff
@@ -28,6 +28,10 @@
     </dependencyManagement>
 
     <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-actuator</artifactId>
```
Comment (meltsufin) [inline spring-cloud-gcp-samples/spring-cloud-gcp-metrics-sample/pom.xml]: @ttomsu When running from IntelliJ, removing it didn't break the sample for me, but now I see that it's actually required. In fact, I think it needs to be moved int our `spring-cloud-gcp-starter-metrics`.

**Comment by meltsufin** [comment]
I moved all changes unrelated to re-enabling the test to a separate PR. See: #121

**Pull request merged**